src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -325,6 +325,175 @@ std::optional<Values> Visitor::visit(Parser::ValuesClauseContext* ctx) {
   return visitOptional(ctx->dataBlock());
 }
 
+// ____________________________________________________________________________________
+ParsedQuery Visitor::visit(Parser::UpdateContext* ctx) {
+  visit(ctx->prologue());
+
+  if (ctx->update()) {
+    reportNotSupported(ctx->update(),
+                       "Multiple updates in one query are not supported.");
```

Comment:
```suggestion
                       "Multiple updates in one query are");
```
(the`reportNotSuppoted` automatically adds "... currently not supported by QLever").
The "are/is" is required for grammatical correctness).